### PR TITLE
feat(content): Innawoods

### PR DESCRIPTION
**Pull request by Zlorthishen**
## Purpose of change
Adds a BN version of `innawoods`, clean-room reverse engineered with no input from the dda version

## Describe the solution
- blacklists most starting scenarios
- removes roads
- removes almost all `specials`

## Testing
~~works fine as long as `CITY_SIZE` = 0~~
towns will still appear if `CITY_SIZE` > 0, however it only consists of a road intersection and nothing else.

## Additional context
this mod does not include any of the bushcraft elements of the dda port
this mod is just to blacklist cities and urban locations, and all scenarios except `Wilderness`

### CODE REVIEW

**Review by chaosvolt — CHANGES_REQUESTED** [review]
Okay so...it relies a lot on full overrides for things like overmap terrain, region settings, and even scenarios. In fact the scenario override especially is really bad.

If it breaks when city size is above zero in the first place then just leave scenarios untouched, scenarios with the `CITY_START` flag are already automatically excluded in zero-city-size worlds.

**Comment by chaosvolt** [comment]
Re-stating it here: the full scenario overrides are not nessecary and will just break compatibility with other mods.

**Review by chaosvolt — CHANGES_REQUESTED** [review]
And now you have full overrides of overmap specials instead of using `copy-from`?

File: data/mods/innawoods/specials.json [hunk 1]
```diff
@@ -0,0 +1,1545 @@
...
+  },
+  {
+    "type": "overmap_special",
+    "id": "lake_shore_dock_small",
+    "copy-from": "lake_shore_dock_small",
+    "occurrences": [ 0, 0 ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "trailhead_basic",
```
Comment (chaosvolt) [inline data/mods/innawoods/specials.json:911]: The continued saga of large numbers of full overrides to overmap specials still polluting this mod.

**Review by chaosvolt — CHANGES_REQUESTED** [review]

File: data/mods/innawoods/specials.json (hunk 1 above)
Comment (Lamandus) [reply data/mods/innawoods/specials.json:911]: what @chaosvolt is trying to say is the following mantra:

The less lines you have to change, the better. Using "copy from" would be a good starting point. Try to recycle as much as you can.

**Review by chaosvolt — APPROVED** [review]
Okay, I think...hope, it's ready...

**Pull request merged**